src/components/KeyValueEditor/KeyValueEditor.tsx

```diff
@@ -1,18 +1,30 @@
-import React, { FunctionComponent, ReactNode, useState } from 'react';
+import React, { FunctionComponent, ReactNode, useEffect, useState } from 'react';
 
+import { useTableSort } from '../../hooks/useTableSort';
 import { DefaultProps } from '../../types';
+import { Pagination } from '../Pagination/Pagination';
 import { Table } from '../Table/Table';
-import { TextInput } from '../TextInput/TextInput';
+import { TextArea } from '../TextArea/TextArea';
 
+import { Form, FormGroup, Spacer, TextInput, Toolbar, ToolbarItem, ToolbarRight } from '../..';
```

Comment:
Circular dependencies vermijden. Hier import je components uit een barrel file waar je de huidige component ook in export.